server/wfnews-api/wfnews-api-rest-endpoints/src/main/java/ca/bc/gov/nrs/wfnews/api/rest/v1/spring/SecuritySpringConfig.java

```diff
@@ -143,9 +143,9 @@ protected void configure(HttpSecurity http) throws Exception {
   @Bean
   public CorsConfigurationSource corsConfigurationSource() {
     CorsConfiguration configuration = new CorsConfiguration();
-		configuration.addAllowedOrigin("*");
-		configuration.addAllowedMethod("*");
-		configuration.addAllowedHeader("*");
+		configuration.addAllowedOrigin(Arrays.asList("https://wildfiresituation.nrs.gov.bc.ca/", "https://wfnews-client.dev.bcwildfireservices.com/", "https://wfnews-client.test.bcwildfireservices.com/"));
```

Comment:
We might want to add these without the trailing backslash also, if not exactly matching will cause this to fail. I notice that the Origin in requests on Chrome and Firefox is `https://wfnews-client.dev.bcwildfireservices.com`